lib/lotuslog/levels.go

```diff
@@ -14,5 +14,7 @@ func SetupLogLevels() {
 		logging.SetLogLevel("bitswap", "WARN")
 		//logging.SetLogLevel("pubsub", "WARN")
 		logging.SetLogLevel("connmgr", "WARN")
+		logging.SetLogLevel("advmgr", "DEBUG")
+		logging.SetLogLevel("stores", "DEBUG")
```

Comment:
Do we want it at debug or was it only for debugging?

Reply:
Let's keep it for now